cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,66 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it(`should have 'conduit' link`, () => {
```

Comment:
https://monosnap.com/file/DIxXoXe5xivmIAMsI7YZCwSqVnloQJ
pay attention to the task. I see many unnecessary tests in your work